com.unity.render-pipelines.universal/Shaders/SpatialMappingOcclusion.shader

```diff
@@ -0,0 +1,72 @@
+Shader "Universal Render Pipeline/SpatialMappingOcclusion"
+{
+    SubShader
+    {
+        Tags {"RenderType" = "Opaque" "IgnoreProjector" = "False" "RenderPipeline" = "UniversalPipeline" }
```

Comment:
Looking at the original shader the queue is geometry-1, this would make sure it is rendered before all other materials, this should be added here.